src/store/reducers/index.js

```diff
@@ -1,12 +1,15 @@
 import { combineReducers } from "redux";
 import { connectRouter } from 'connected-react-router'
 import createReturn from "./createReturn"
+
+import belReducer from './belReducer'
```

Comment:
Please rename belReducer so that it relates to the function that it is performing.
If belReducer isn't being used any more, please remove this line.